omnigibson/objects/object_base.py

```diff
@@ -192,6 +192,9 @@ def _post_load(self):
             self.root_prim.RemoveAPI(lazy.pxr.UsdPhysics.ArticulationRootAPI)
             self.root_prim.RemoveAPI(lazy.pxr.PhysxSchema.PhysxArticulationAPI)
 
+        if og.sim.is_playing:
+            log.warning("Cannot set articulation root API while simulation is playing!")
```

Comment:
hmm interesting.. are you sure? Previously I was able to import new object while sim is playing. Are you suggesting the articulation root API won't work?

Reply:
This doesn't work all the time. It depends on how the articulation root is annotated originally. Essentially the reason why we added this warning is we can only load some robots while sim is playing but not the others.

Reply:
This doesn't work all the time. It depends on how the articulation root is annotated originally. Essentially the reason why we added this warning is we can only load some robots while sim is playing but not the others.

Reply:
@ChengshuLi this causes a very annoying problem: when you import a USD while the simulation is playing, the object gets loaded into PhysX before post_load can run, meaning it gets loaded with the articulation root API still on w/e prim it was on in the USD regardless of whether the USD is fixed or not, and you can't fix it without stopping (e.g. even if you move the articulation root API on the USD, physx remains unaware). We can hack this by removing the articulation root API from all of our USDs, then at load time we can pre-load it using the Usd library instead of loading it directly into Isaac, add the articulation root, and load that one into Isaac instead - I'll do it with the next asset release.